### PR TITLE
expanded RPKG GUI section

### PR DESCRIPTION
**Pull request by nirwanah**

### CODE REVIEW

File: docs/rpkg/gui/opening and finding files.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 2
+---
+
+# Opening RPKG's
```
Comment (RDIL) [inline docs/rpkg/gui/opening and finding files.md]:
```suggestion
# Opening RPKGs
```
Not possessive.

File: docs/rpkg/gui/opening and finding files.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Once setup you can load in the chunk files into the tool by using `Import` > `Import RPKGs folder` and pointing it to the `HITMAN 3\Runtime` folder. This will load all the chunk files into the tool which is recommended when you are unsure where the file you are looking for is located. You can also load in individual files using the `Import` > `Import RPKG file`
+
+# Searching RPKG's
```
Comment (RDIL) [inline docs/rpkg/gui/opening and finding files.md]:
```suggestion
# Searching RPKGs
```
Same here.

File: docs/rpkg/gui/opening and finding files.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+You can search for files using the Search tab in the left panel. Search can be done based on hash in the `Seach Hash List` tab or on filename in the `Search RPKGs` tab. The search will only search in the currently loaded RPKGs. Wildcards are not permitted.  
+
+
+
+The wiki is currently a work in progress at the moment, please feel free to contribute @ https://github.com/glacier-modding/wiki.notex.app
```
Comment (RDIL) [inline docs/rpkg/gui/opening and finding files.md:13]: Presumably, this comment is made obsolete by this PR.

**Review by RDIL — CHANGES_REQUESTED** [review]
This issue is present a number of times.

File: docs/rpkg/gui/opening and finding files.md (hunk 3 above)
Comment (nirwanah) [reply docs/rpkg/gui/opening and finding files.md:13]: I'd say leave it for now in case anyone has something to add we might have overlooked.

**Review by RDIL — APPROVED** [review]

**Pull request merged**